package.json

```diff
@@ -28,26 +28,26 @@
     "test:coverage": "NODE_ENV=test jest --detectOpenHandles --forceExit --coverage --coverageDirectory ./app/tests/coverage"
   },
   "dependencies": {
-    "dotenv": "^10.0.0",
-    "javascript-time-ago": "^2.3.8"
+    "dotenv": "^16.0.1",
+    "javascript-time-ago": "^2.5.7"
   },
   "devDependencies": {
-    "@google-cloud/functions-framework": "^1.9.0",
-    "@vuepress/plugin-back-to-top": "^1.8.2",
-    "@vuepress/plugin-pwa": "^1.8.2",
-    "babel-eslint": "^10.1.0",
-    "eslint": "^7.32.0",
-    "eslint-config-airbnb-base": "^14.2.1",
-    "eslint-plugin-import": "^2.24.2",
-    "eslint-plugin-jest": "^24.4.0",
-    "eslint-plugin-jsdoc": "^36.0.8",
-    "eslint-plugin-vue": "^7.17.0",
-    "husky": "^7.0.2",
-    "jest": "^27.1.0",
-    "nock": "^13.1.3",
-    "typescript": "^4.4.2",
-    "vuepress": "^1.8.2",
-    "vue-eslint-parser": "^7.10.0"
+    "@babel/eslint-parser": "^7.18.9",
+    "@google-cloud/functions-framework": "^3.1.2",
+    "@vuepress/plugin-back-to-top": "^1.9.7",
+    "@vuepress/plugin-pwa": "^1.9.7",
+    "eslint": "^8.22.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^26.8.3",
+    "eslint-plugin-jsdoc": "^39.3.6",
+    "eslint-plugin-vue": "^9.3.0",
+    "husky": "^8.0.1",
+    "jest": "^28.1.3",
+    "nock": "^13.2.9",
+    "typescript": "^4.7.4",
+    "vue-eslint-parser": "^9.0.3",
+    "vuepress": "^1.9.7"
```

Comment:
VuePress is now in maintenance mode (see: https://github.com/vuejs/vuepress#status), and is causing ~21 vulnerabilities (2 low, 8 moderate, 11 high).

We could consider to switch to VitePress (https://vitepress.vuejs.org/) but it's still an alpha version.